### ISSUE TITLE
Error saving settings: 'Name'

### ISSUE THREAD

**Issue by zmija** [Oxygem/Kanmail#111]
When I try setting up a new account using IMAP and TLS encryption the connection seems to be established, but after a click on a save button I receive "Error saving settings: 'Name'" error:

![image](https://user-images.githubusercontent.com/8327569/81101062-d6d74380-8f0d-11ea-8c1a-ca724a51dce5.png)

In the logs (?) on top right, I receive the following error: 

![image](https://user-images.githubusercontent.com/8327569/81101107-ebb3d700-8f0d-11ea-8730-69e5de96a2a2.png)

If I click on the log (as it's suggested in the log message), all the data disappears and I have to full fill the form again. Tried 3 times now and I'm clueless what's wrong. Any suggestions?

---

I love the idea and I really hope the product will meet the expectations. I'm tech savvy but I find the setup screen very confusing (including the error msg above).

**Comment by Fizzadar** [comment]
@zmija these errors are not helpful! The latest release (`v1.2005061610`) should display more information and the full traceback when encountering this. If you wouldn't mind testing with that and pasting the error log back here (I've tried and failed to replicate this).

**Comment by Fizzadar** [comment]
@zmija I have just figured this out! The account's all need a name field (top of the new account form).

I'm going to leave this issue open to improve the error handling here, as it's awful!

**Comment by zmija** [comment]
hi @Fizzadar, thanks for your reply.

I added a name field on top. Name can be anything, right? Or is there any validation?
Yesterday I put the same name as the username (so type email) and today I put just a string "Mia" but I got the same error.

I forgot to mention I'm running this on macOS Catalina 10.15.4.

I'm pasting additional logs I can see now in the latest version, hope it can be of any help:

```
Status: 500
Traceback (most recent call last):
  File "site-packages/flask/app.py", line 1813, in full_dispatch_request
  File "site-packages/flask/app.py", line 1799, in dispatch_request
  File "kanmail/server/views/settings_api.py", line 38, in api_set_settings
  File "kanmail/settings/model.py", line 99, in validate_unique_accounts
KeyError: 'name'
```

Also, the current email address I'm trying to link with Kanmail is linked with Sogo which is also not an ideal email client, so it's very possible the issue is on their side.
